src/lib/chat/matrix/chat-message.ts

```diff
@@ -58,6 +58,7 @@ export async function mapMatrixMessage(matrixMessage, sdkMatrixClient: SDKMatrix
     },
     parentMessageText: '',
     parentMessageId: parent ? parent['m.in_reply_to']?.event_id : null,
+    isHidden: content?.msgtype === MatrixConstants.BAD_ENCRYPTED_MSGTYPE,
```

Comment:
I'm guessing all message related stuff goes through here :crossed_fingers: but I think there's some special logic for message edit events. Did you happen to verify that edit events that we can't decrypt end up with this message? I seem to recall some specialized logic that perhaps we can change to this now. :thinking:

Reply:
yes there's a different logic for capturing the edit events (from the code i am seeing). I had a doubt if we want to display the same message on that event as well. I have asked on the JIRA task, will update in followUp as required. 

Currently there is no change of behaviour with message edit.